scripting/include/srccoop/classdef.inc

```diff
@@ -927,8 +1052,17 @@ methodmap CBasePlayer < CBaseCombatCharacter
 		CBaseAnimating pItem = CBaseAnimating(GivePlayerItem(this.GetEntIndex(), szItemName[iNameOffset]));
 		if(pItem.IsValid() && !pItem.GetOwner().IsValid())
 		{
-			// this would drop on the ground if player cant pick up any more
-			pItem.KillAfterTime(0.1);
+			if (strcmp(szItemName, "item_suit", false) == 0)
```

Comment:
In theory we could just SetSuit(true) without spawning anything.

Reply:
I had tested that, but this appears to be too soon, or potentially another function is removing the suit immediately afterwards.
I haven't fully tested or found where it is being removed from, but it would be more ideal to be able to just SetSuit(true).

Reply:
I have changed the order for equipping the player in `Hook_PlayerSpawnPost` and changed the `GiveItem` of `item_suit` to just SetSuit(true) and return null entity instead of creating at all.